.gitlab-ci.yml

```diff
@@ -0,0 +1,12 @@
+stages:
```

Comment:
Why you created gitlab-ci configuration? It should be git-hub action configuration

Reply:
I actually have both :)
I am using gitlab and github at the same time. Was wondering how this feature works with multi remote in Intellij Idea